### DISCUSSION TITLE
How can I share to a specific link?

### DISCUSSION THREAD

**Discussion by sassanix** [FreshRSS/FreshRSS#6635] [General]
Hi everyone,

I'm trying to use archive.is instead of archive.ph , and I can't add customized links the way I want to set them.

How should I go about it?

**Comment by math-GH** [comment]
Currently there is no way for customized sharing services.

Why do want to use archive.is instead of archive.ph? Isn't it the same service?

**Reply by sassanix** [reply]
Archive.ph is blocked at work and some other places but archive.is isn't.

**Comment by math-GH** [comment]
I created a PR to add this domain too. See #6650

**Comment by sassanix** [comment]
> I created a PR to add this domain too. See #6650

Thank you so much!

**Reply by math-GH** [reply]
It is now in the `edge` branch and will be delivered with 1.24.2 (no release date yet)

**Reply by sassanix** [reply]
> It is now in the `edge` branch and will be delivered with 1.24.2 (no release date yet)

I'll wait for the release, thank you for developing freshrss! I tell everyone about it 😄
